### PR TITLE
Webpack compatibility 

### PR DESCRIPTION
**Pull request by bassettsj**
While trying to work on using bunyan for a universal JS app using react / redux, and express, I found problems trying to bundle bunyan using webpack. I thought I would add a unit test showing the issues for now and maybe work on fixing those issues.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,15 +7,21 @@
   "bin": {
     "bunyan": "./bin/bunyan"
   },
-
```
Comment (bassettsj) [inline package.json]: NPM seems to have reformtted a lot of the package.json with `npm i -D webpack`, should I revert these changes.

**Comment by amejias101** [comment]
@bassettsj is bunyan bundling for you?

**Comment by trentm** [comment]
@bassettsj Thanks for this. I realize it has been over *a year* now. I tried this rebased on the current Bunyan master, and I get this failure:

```
[19:53:25 trentm@danger0:~/tm/node-bunyan (pr-408)]
$ ./node_modules/.bin/nodeunit test/webpack.test.js

webpack.test.js
✖ webpack

Assertion Message: stats.hasErrors() should return false
AssertionError: true == false
    at Object.equal (/Users/trentm/tm/node-bunyan/node_modules/nodeunit/lib/types.js:83:39)
    at /Users/trentm/tm/node-bunyan/test/webpack.test.js:26:11
    at Compiler.<anonymous> (/Users/trentm/tm/node-bunyan/node_modules/webpack/lib/Compiler.js:194:14)
    at Compiler.emitRecords (/Users/trentm/tm/node-bunyan/node_modules/webpack/lib/Compiler.js:282:37)
    at Compiler.<anonymous> (/Users/trentm/tm/node-bunyan/node_modules/webpack/lib/Compiler.js:187:11)
    at /Users/trentm/tm/node-bunyan/node_modules/webpack/lib/Compiler.js:275:11
    at Compiler.applyPluginsAsync (/Users/trentm/tm/node-bunyan/node_modules/webpack/node_modules/tapable/lib/Tapable.js:60:69)
    at Compiler.afterEmit (/Users/trentm/tm/node-bunyan/node_modules/webpack/lib/Compiler.js:272:8)
    at Compiler.<anonymous> (/Users/trentm/tm/node-bunyan/node_modules/webpack/lib/Compiler.js:267:14)
    at /Users/trentm/tm/node-bunyan/node_modules/webpack/node_modules/async/lib/async.js:52:16
    at done (/Users/trentm/tm/node-bunyan/node_modules/webpack/node_modules/async/lib/async.js:246:17)
    at /Users/trentm/tm/node-bunyan/node_modules/webpack/node_modules/async/lib/async.js:44:16
    at /Users/trentm/tm/node-bunyan/node_modules/webpack/node_modules/enhanced-resolve/node_modules/graceful-fs/graceful-fs.js:43:10
    at FSReqWrap.oncomplete (fs.js:82:15)


FAILURES: 1/2 assertions failed (850ms)
```

If you are so inclined, I'd appreciate help on that. I don't know webpack at all.

**Comment by bassettsj** [comment]
@trentm I haven't tried in a while, but I think it is still broken. Webpack will try to load all the requires before bundling, unlike browserify, so it causes it to fail. I can give this a try later, but I haven't really been paying much attention to it late. :-/
